app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Api::V1::BooksController < ApplicationController
+
+  #Get /books
+  def index
+    @books = Book.all
```

Comment:
there's no need to use @books, because we're not using rails views.
Also where's the paging and filtering for books